tests.py

```diff
@@ -20,53 +21,36 @@ def test_cage(self):
         cage = Cage()
```

Comment:
It's the one for testing a cage, but cage doesn't have anything in it. Just exists.
Not sure how can I test it with assertions